HW python.py

```diff
@@ -0,0 +1,16 @@
+#my code:
+
+with open('text.txt') as file:
```

Comment:
with...open... context should only include file operation

Reply:
@AviadP I changed the file mode - read mode

Reply:
pls use "with..open..." context manager, as explained in class

Reply:
@AviadP  i change at the new file to - with open('text.txt', 'r') as file